helm/adelphi/templates/nosqlbench-jobs.yaml

```diff
@@ -39,7 +41,7 @@ spec:
       command: ["sh", "-c", "-e"]
       args:
         - pip install adelphi;
-          adelphi --hosts="{{ .Values.source.clusterName }}-{{ .Values.source.dc }}-service" --username="{{ .Values.source.clusterName }}-superuser" --password="`cat /secret/password`" --keyspaces="{{ .Values.gemini_schema_keyspace }}" --no-anonymize export-nb --rampup-cycles="{{ .Values.nosqlbench_cycles }}" --main-cycles="{{ .Values.nosqlbench_cycles }}" > /workspace/nosqlbench_source.yaml;
+          adelphi --hosts="{{ .Values.source.clusterName }}-{{ .Values.source.dc }}-service" --username="{{ .Values.source.clusterName }}-superuser" --password="`cat /secret/password`" --keyspaces="{{ .Values.gemini_schema_keyspace }}" --no-anonymize export-nb > /workspace/nosqlbench_source.yaml;
```

Comment:
So, I get the change here, but I think I probably need to understand a bit more about what the python tooling spits out for the template.

Maybe and first question... why were these two parameters provided for within the python tooling to begin with?  And why is not having them there the "right" way to do this?

Reply:
The goal here is to separate execution parameters from the template generation parameters.
The number of cycles is something that matters only at runtime, so once you have the workload YAML generated, you can tweak the number of cycles as you wish without generating the YAML all over again (which requires a connection to the cluster).

Another benefit is keeping the adelphi cli slim and relying on the existing variable replacement feature in NoSQLBench instead.